.php_cs

```diff
@@ -1,29 +1,28 @@
 #!/usr/bin/env php
 <?php
 
-$finder = Symfony\CS\Finder::create()
-    ->ignoreDotFiles(true)
-    ->ignoreVCS(true)
-    ->exclude('app/config')
-    ->exclude('app/data')
-    ->exclude('app/Resources')
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
     ->exclude('var')
     ->exclude('vendor')
-    ->exclude('web/bundles')
-    ->exclude('web/css')
-    ->exclude('web/fonts')
-    ->exclude('web/js')
     ->notPath('web/config.php')
-    ->in(__DIR__)
+    // Doctrine fixtures are excluded to avoid these PHP-CS-Fixer errors:
+    // "Files were not fixed due to errors reported during linting before fixing"
+    ->notPath('DataFixtures')
 ;
 
-return Symfony\CS\Config::create()
-    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
-    ->fixers([
-        '-psr0', // Ignore Tests\ namespace prefix mismatch with tests/ directory
-        'ordered_use',
-        'phpdoc_order',
-        'short_array_syntax',
+return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@Symfony' => true,
+        '@Symfony:risky' => true,
+        'array_syntax' => ['syntax' => 'short'],
+        'no_useless_else' => true,
+        'no_useless_return' => true,
+        'ordered_imports' => true,
+        'phpdoc_order' => true,
+        'php_unit_strict' => true,
+        'strict_comparison' => true,
```

Comment:
well, Symfony itself decided to order new use statements only, due to maintenance constraints. But this does not forbid this demo project to be strict about them.
The same is true for the short array syntax. This demo decided to use it.

regarding ``no_useless_else``, if it does what I think it does according to its name, it should be part of the ruleset, as Symfony indeed mandates that we don't use ``else`` when the ``if`` quits the execution path (through ``return``, ``throw``, ``continue`` or ``break``